cmd/ctr/commands/images/export.go

```diff
@@ -21,36 +21,33 @@ import (
 	"os"
 
 	"github.com/containerd/containerd/cmd/ctr/commands"
-	"github.com/containerd/containerd/images/oci"
-	"github.com/containerd/containerd/reference"
-	digest "github.com/opencontainers/go-digest"
+	"github.com/containerd/containerd/images/archive"
+	"github.com/containerd/containerd/platforms"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
 	"github.com/urfave/cli"
 )
 
 var exportCommand = cli.Command{
 	Name:      "export",
-	Usage:     "export an image",
-	ArgsUsage: "[flags] <out> <image>",
-	Description: `Export an image to a tar stream.
-Currently, only OCI format is supported.
+	Usage:     "export images",
+	ArgsUsage: "[flags] <out> <image> ...",
+	Description: `Export images to an OCI tar archive.
+
+Tar output is formatted as an OCI archive, a Docker manifest is provided for the platform.
+Use '--skip-manifest-json' to avoid including the Docker manifest.json file.
+Use '--platform' to define the output platform.
+When '--all-platforms' is given all images in a manifest list must be available.
 `,
 	Flags: []cli.Flag{
-		// TODO(AkihiroSuda): make this map[string]string as in moby/moby#33355?
-		cli.StringFlag{
-			Name:  "oci-ref-name",
-			Value: "",
-			Usage: "override org.opencontainers.image.ref.name annotation",
```

Comment:
why not keep this

Reply:
The problem is this annotation is so poorly defined that is unclear what the correct behavior is when outputting multiple images. I agree we should solve this in a follow up to make sure we come up with a good solution for generating OCI indexes in a sane manner when multiple images are involved.